index.php

```diff
@@ -64,24 +104,26 @@
         </div>
         <ul class="lots__list">
             <!--заполните этот список из массива с товарами-->
+            <?php foreach ($lots as $key => $val): ?>
```

Comment:
Ключи тут не нужны, так что можно убрать `$key =>`.
Второй момент: `val` лучше заменить на `lot`, чтобы по названию переменной было понятно что это.
```suggestion
            <?php foreach ($lots as $lot): ?>
```